### PR TITLE
Add osu! South East Asia Tournament 4 (oSEAT4) wiki article

### PR DESCRIPTION
**Pull request by Nivalyx**
wiki article for the currently ongoing osu! South East Asia Tournament 4

### CODE REVIEW

File: wiki/Tournaments/oSEAT/4/en.md [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+tags:
+  - oSEAT
+  - oSEAT4
+---
+
+# osu! South East Asia Tournament 4
+
+![oSEAT4 logo](img/logo.jpg)
+
+The **osu! South East Asia Tournament 4** (***oSEAT4***) is a double-elimination 1v1 osu!standard tournament hosted by ![][flag_SG] [phox](https://osu.ppy.sh/users/772295). The tournament is open to all players from all ten [ASEAN member states](https://asean.org/asean/asean-member-states/) (![][flag_BN] Brunei Darussalam, ![][flag_KH] Cambodia, ![flag_ID] Indonesia, ![flag_LA] Laos, ![flag_MY] Malaysia, ![flag_MM] Myanmar, ![flag_PH] The Phillipines, ![flag_SG] Singapore, ![flag_TH] Thailand, and ![flag_VN] Vietnam) regardless of rank. It is the fourth iteration of the tournament series.
```
Comment (TPGPL) [inline wiki/Tournaments/oSEAT/4/en.md:11]:
```suggestion
The **osu! South East Asia Tournament 4** (***oSEAT4***) is a double-elimination 1v1 osu! tournament hosted by ![][flag_SG] [phox](https://osu.ppy.sh/users/772295). The tournament is open to all players from all ten [ASEAN member states](https://asean.org/asean/asean-member-states/) (![][flag_BN] Brunei Darussalam, ![][flag_KH] Cambodia, ![flag_ID] Indonesia, ![flag_LA] Laos, ![flag_MY] Malaysia, ![flag_MM] Myanmar, ![flag_PH] The Phillipines, ![flag_SG] Singapore, ![flag_TH] Thailand, and ![flag_VN] Vietnam) regardless of rank. It is the 4th installment of the osu! South East Asia Tournament.
```

File: wiki/Tournaments/oSEAT/4/en.md [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2020-11-07/2020-11-22 |
+| Screening phase | 2020-11-23/2020-11-29 |
+| Qualifiers | 2020-11-30/2020-12-06 |
+| Group stage (week 1) | 2020-12-07/2020-12-13 |
+| Group stage (week 2) | 2020-12-14/2020-12-20 |
```
Comment (TPGPL) [inline wiki/Tournaments/oSEAT/4/en.md:21]:
```suggestion
| Group Stage week 1 | 2020-12-07/2020-12-13 |
| Group Stage week 2 | 2020-12-14/2020-12-20 |
```

File: wiki/Tournaments/oSEAT/4/en.md [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+| Round of 32 | 2020-12-21/2020-12-27 |
+| Round of 16 | 2020-12-28/2021-01-03 |
+| Quarterfinals | 2021-01-04/2021-01-10 |
+| Semifinals | 2021-01-11/2021-01-17 |
+| Finals (week 1) | 2021-01-18/2021-01-23 |
+| Finals (week 2) | 2021-01-24/2021-01-30 |
```
Comment (TPGPL) [inline wiki/Tournaments/oSEAT/4/en.md:27]:
```suggestion
| Finals week 1 | 2021-01-18/2021-01-23 |
| Finals week 2 | 2021-01-24/2021-01-30 |
```

File: wiki/Tournaments/oSEAT/4/en.md [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 12 months of osu!supporter tag, unique profile badge |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 6 months of osu!supporter tag |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 4 months of osu!supporter tag |
+| 4th place | 2 months of osu!supporter tag |
+| 5th place | 1 month of osu!supporter tag |
```
Comment (TPGPL) [inline wiki/Tournaments/oSEAT/4/en.md:37]:
```suggestion
| *4th place* | 2 months of osu!supporter tag |
| *5th place* | 1 month of osu!supporter tag |
```

File: wiki/Tournaments/oSEAT/4/en.md [hunk 5]
```diff
@@ -0,0 +1,346 @@
...
+
+![Profile badge](img/Badge.png)
```
Comment (TPGPL) [inline wiki/Tournaments/oSEAT/4/en.md:39]:
```suggestion
![](img/Badge.png "oSEAT4 winner badge")
```

File: wiki/Tournaments/oSEAT/4/en.md [hunk 6]
```diff
@@ -0,0 +1,346 @@
...
+
+## Organization
+
+oSEAT4 was run by various osu! community members predominantly hailing from South East Asian countries.
```
Comment (TPGPL) [inline wiki/Tournaments/oSEAT/4/en.md:43]:
```suggestion
oSEAT4 is run by various osu! community members predominantly hailing from South East Asian countries.
```

File: wiki/Tournaments/oSEAT/4/en.md [hunk 7]
```diff
@@ -0,0 +1,346 @@
...
+
+| Position | Members |
```
Comment (TPGPL) [inline wiki/Tournaments/oSEAT/4/en.md:45]:
```suggestion
| Position | Member(s) |
```

File: wiki/Tournaments/oSEAT/4/en.md [hunk 8]
```diff
@@ -0,0 +1,346 @@
...
+| :-- | :-- |
+| Host | ![][flag_SG] [phox](https://osu.ppy.sh/users/772295) |
+| Map selector | ![][flag_SG] [phox](https://osu.ppy.sh/users/772295), ![][flag_SG] [Fragmented](https://osu.ppy.sh/users/847434), ![][flag_TH] [Seaweed](https://osu.ppy.sh/users/5151214), ![][flag_SG] [Ayucchi](https://osu.ppy.sh/users/7823498), ![][flag_US] [Not A Function](https://osu.ppy.sh/users/4773094) |
```
Comment (TPGPL) [inline wiki/Tournaments/oSEAT/4/en.md:48]:
```suggestion
| Mappool selector | ![][flag_SG] [phox](https://osu.ppy.sh/users/772295), ![][flag_SG] [Fragmented](https://osu.ppy.sh/users/847434), ![][flag_TH] [Seaweed](https://osu.ppy.sh/users/5151214), ![][flag_SG] [Ayucchi](https://osu.ppy.sh/users/7823498), ![][flag_US] [Not A Function](https://osu.ppy.sh/users/4773094) |
```

File: wiki/Tournaments/oSEAT/4/en.md [hunk 9]
```diff
@@ -0,0 +1,346 @@
...
+- **[Main spreadsheet](https://docs.google.com/spreadsheets/d/16LDQYYDwd7obzSgLijTV1hBD7QOZE_2Ibe9ZLqTWQNw/edit#gid=1373445856)**
+- [Forum thread](https://osu.ppy.sh/community/forums/topics/1177376)
+- [Discord server](https://discord.gg/sVHndPU)
+- [Livestream channel](https://www.twitch.tv/oseatournament)
+
+## Participants
+
+Listed below are players who qualified into the Group stage (along with their respective seeding and group placement) out of 179 registered players in total.
+
+| Group | Player 1 (Seed A) | Player 2 (Seed B) | Player 3 (Seed C) | Player 4 (Seed D) | Player 5 (Seed E) | Player 6 (Seed F) |
```
Comment (TPGPL) [inline wiki/Tournaments/oSEAT/4/en.md:68]:
```suggestion
| Group | Seed A | Seed B | Seed C | Seed D | Seed E | Seed F |
```
should be fine this way

File: wiki/Tournaments/oSEAT/4/en.md [hunk 10]
```diff
@@ -0,0 +1,346 @@
...
+
+¹ *![][flag_ID] [ngonk](https://osu.ppy.sh/users/12030070) initially qualified for this Group stage berth, but they decided to forfeit from the tournament during the running of the Group stage. Under the discretion of staff members and all other affected players, ![][flag_MY] [Heya](https://osu.ppy.sh/users/11379332) was later admitted to the Group stage in their spot.*
+
+² *![][flag_ID] [cfood](https://osu.ppy.sh/users/8626190) initially qualified for this Group stage berth, but they decided to forfeit from the tournament during the running of the Group stage. Under the discretion of staff members and all other affected players, ![][flag_PH] [Revillica](https://osu.ppy.sh/users/9806095) was later admitted to the Group stage in their spot.*
+
+³ *Despite having the ![][flag_US] United States flag displayed on their profile, [\_Kolin](https://osu.ppy.sh/users/7249644) is eligible to participate in the tournament as they are in possession of a valid ![][flag_PH] [Phillipine passport](https://en.wikipedia.org/wiki/Philippine_passport).*
+
+## Mappools
+
+### Group stage (week 2)
```
Comment (TPGPL) [inline wiki/Tournaments/oSEAT/4/en.md:95]:
```suggestion
### Group Stage week 2
```

File: wiki/Tournaments/oSEAT/4/en.md [hunk 11]
```diff
@@ -0,0 +1,346 @@
...
+  - [Grand Thaw - SOLROS (Lunala) \[Fantasia\]](https://osu.ppy.sh/beatmapsets/1136592#osu/2376019)
+  - [siromaru - Absurd Gaff (spro) \[Another\]](https://osu.ppy.sh/beatmapsets/17563#osu/66662)
+- DoubleTime
+  - [Shirakami Fubuki - Say! Fanfare! (Bokamin) \[Insane\]](https://osu.ppy.sh/beatmapsets/1277448#osu/2658704)
+  - [Giga-P - Electric Angel (koreapenguin) \[Sonnyc's Insane\]](https://osu.ppy.sh/beatmapsets/172900#osu/417898)
+- FreeMod
+  - [M2U - A Bella! (feat. Lucy) (Okoratu) \[Battle's Extra\]](https://osu.ppy.sh/beatmapsets/1158767#osu/2418106)
+  - [M2U - Memory of Beach (milr\_) \[Mx\]](https://osu.ppy.sh/beatmapsets/1120861#osu/2342057)
+
+### Group stage (week 1)
```
Comment (TPGPL) [inline wiki/Tournaments/oSEAT/4/en.md:118]:
```suggestion
### Group Stage week 1
```

File: wiki/Tournaments/oSEAT/4/en.md [hunk 12]
```diff
@@ -0,0 +1,346 @@
...
+- HardRock
+  - [Linkin Park - Session (h4d0uk3n1) \[Katherine\]](https://osu.ppy.sh/beatmapsets/669536#osu/1488047)
+  - [Dance Gavin Dance - Tree Village (Dilectus) \[Extra\]](https://osu.ppy.sh/beatmapsets/703666#osu/1488698)
+- DoubleTime
+  - [Buono! - Winter Story (Nakagawa-Kanon) \[Insane\]](https://osu.ppy.sh/beatmapsets/40425#osu/128303)
+  - [dj TAKA - Colors -sasakure.UK Futurelogic Remix- (\_wa) \[Hyper\]](https://osu.ppy.sh/beatmapsets/317439#osu/1319822)
+
+## Match results
+
+### Group stage (week 1)
```
Comment (TPGPL) [inline wiki/Tournaments/oSEAT/4/en.md:161]:
```suggestion
### Group Stage week 1
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Comment by Nivalyx** [comment]
applied everything + fixed the remark check

**Review by TPGPL — APPROVED** [review]

**Pull request merged**